README.md

```diff
@@ -7,7 +7,6 @@
   - [🛠 Built With ](#-built-with-)
     - [Key Features ](#key-features-)
```

Comment:
- [ ] Please there should be a verticla border between the action links and buttons as in the design.
- [ ] Please the color of the book card info doesn't match the design the book category should be in light gray color and the author color should be in blue , (please make sure you got the same colors)
![image](https://github.com/jkanyi-web/bookstore-cra/assets/74744787/4e591960-6a08-452b-9537-8892a6a9a75c)